Dockerfile.standalone

```diff
@@ -12,17 +12,33 @@ COPY ./engine ./engine
 COPY ./packages/grafbase-sdk/package.json ./packages/grafbase-sdk
 COPY ./packages/cli-app ./packages/cli-app
 
-RUN apk add --no-cache git musl-dev npm
+RUN apk add --no-cache git musl-dev npm ca-certificates wget
+
+# Bun needs glibc to be installed
+RUN mkdir -p /etc/apk/keys/
+RUN wget -q -O /etc/apk/keys/sgerrand.rsa.pub https://alpine-pkgs.sgerrand.com/sgerrand.rsa.pub
+RUN wget https://github.com/sgerrand/alpine-pkg-glibc/releases/download/2.28-r0/glibc-2.28-r0.apk
+RUN apk add --no-cache --force-overwrite glibc-2.28-r0.apk
 
 WORKDIR /grafbase/packages/cli-app
 
 RUN npx --yes pnpm i
 RUN npx --yes pnpm run cli-app:build
 
+WORKDIR /grafbase/cli/udf-wrapper
+
+RUN npx --yes pnpm i
+RUN npx --yes pnpm run build
+
 WORKDIR /grafbase
 
 RUN cargo build -p grafbase --release
 
+# Remove bun dependencies
+RUN rm -rf /etc/apk/keys/sgerrand.rsa.pub
+RUN apk del glibc
+RUN rm -rf glibc-2.28-r0.apk
```

Comment:
If it's dynamically linked, isn't this going to be a problem?

Reply:
It's only to run bun, we don't need bun with the federated gateway during runtime. Eventually we want to get rid of all the javascript dependencies for the federated gateway. This might mean splitting it to a separate binary, or feature flags.

Reply:
You're removing in both federated and standalone though, no?

Reply:
Yes. Both builds also need bun right now, we need to change that in the future.